library/core/src/mem/maybe_uninit.rs

```diff
@@ -261,6 +269,14 @@ impl<T> MaybeUninit<T> {
     ///
     /// See the [type-level documentation][type] for some examples.
     ///
+    /// # Example
+    ///
+    /// ```
+    /// use std::mem::MaybeUninit;
+    ///
+    /// let v: MaybeUninit<String> = MaybeUninit::uninit();
+    /// ```
+    ///
     /// [type]: union.MaybeUninit.html
```

Comment:
```suggestion
    /// [type]: MaybeUninit
```
I think this should work?